app/utils/data_access.py

```diff
@@ -874,12 +874,28 @@ def add_individual(form, user_uuid):
     if "bodyfat" in form:
         update_bodyfat(individual, form["bodyfat"])
 
-    # TODO: also create herd tracking values
-
     individual.save()
+
+    setattr(individual, "herdtracking_set", True)
```

Comment:
I don't think I follow the logic of adding this - if one don't want to update herd tracking, why not just not call it?

Reply:
I assumed that this was the case. But we can make it a flag of course.